plugins/source/aws/client/spec/account.go

```diff
@@ -0,0 +1,21 @@
+package spec
+
+type Account struct {
+	ID              string   `json:"id" jsonschema:"required,minLength=1"`
+	AccountName     string   `json:"account_name,omitempty"`
+	LocalProfile    string   `json:"local_profile,omitempty"`
+	RoleARN         string   `json:"role_arn,omitempty" jsonschema:"pattern=^arn(:[^:\n]*){5}([:/].*)?$"`
+	RoleSessionName string   `json:"role_session_name,omitempty"`
+	ExternalID      string   `json:"external_id,omitempty"`
+	DefaultRegion   string   `json:"default_region,omitempty" jsonschema:"default=us-east-1"`
```

Comment:
If we are going to set this here. we should clean up the logic for where ever we check if DefaultRegion is an empty string

Reply:
removed default in 2cab8a0